runtime/compiler/control/J9Options.cpp

```diff
@@ -1739,6 +1745,12 @@ J9::Options::fePreProcess(void * base)
                }
             }
          }
+      // Handle -XX:+UseLargePages
+      else if (largePageArgIndex >= 0)
```

Comment:
would it be better `(-1 != largePageArgIndex)` ?

Reply:
Elsewhere in this file they always like to compare `>= 0`. I understand that may be the convention on the VM/GC side, but I suggest we leave it as is for now.